index.html

```diff
@@ -0,0 +1,119 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300&family=Montserrat:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <title>Document</title>
+  </head>
+  <body>
+    <nav id="nav-section" class="grid">
+      <div class="nav-text">
+        <span class="smaller-text">Already have an Intuit account?</span> <br />
+        <span class="smaller-text"
+          ><a href="#">I forgot my User ID or Password</a></span
+        >
+      </div>
+      <button id="sign-in"><i class="fa fa-lock"></i> Sign In</button>
+    </nav>
+
+    <main>
+      <div class="text-center mt-15">
+        <img
+          src="https://cdn.freebiesupply.com/images/large/2x/intuit-logo-transparent.png"
+          alt="intuit logo"
+          width="130"
+          class="m-0-auto"
+        />
+        <div id="icons" class="flex vertical-padding-16">
+          <img
+            src="https://lh3.googleusercontent.com/kAoKkNJy81qdxeSexsfsfH6DEz114px1qx9zg0Hj3RoonLXxXRBbEp0xIZVVElgihQ"
+            alt="mint-logo"
+            width="25"
+          />
+          <p class="montserrat">mint</p>
+          <img
+            src="https://www.forecast.app/hubfs/quickbooks.svg"
+            alt="mint-logo"
+            width="25"
+          />
+          <p class="montserrat">quickbooks</p>
+          <img
+            src="http://getdrawings.com/free-icon/quicken-icon-73.png"
+            alt="mint-logo"
+            width="25"
+          />
+          <p class="montserrat">turbotax</p>
+        </div>
+      </div>
+      <div id="border-box">
+        <form class="dark-gray">
+          <h2 class="text-center dark-gray">Create an Intuit account</h2>
+          <p class="text-center small-medium-text">
+            One account for everything Intuit, including Mint.
+          </p>
+          <p class="text-center smaller-text"><a href="">Learn more</a></p>
+          <br />
+          <div class="vertical-padding-16">
+            <label for="">Email address</label>
+            <input class="mt-10" type="text" id="" name="" />
+          </div>
+          <div class="vertical-padding-16">
+            <label for="">Phone (recommended)</label>
+            <input class="mt-10" type="text" id="" name="" />
+            <p class="mt-5 x-small-text">
+              Standard call, messaging or data rates may apply
+            </p>
+          </div>
+          <div class="vertical-padding-16">
+            <label for="">Password</label>
+            <input class="mt-10" type="text" id="" name="" />
+            <i class="fa fa-unlock input-icon"></i>
+          </div>
+          <button id="submit-btn" class="larger-text">
```

Comment:
PLease fix the type of the input and buttons:
- [ ] It has an `<input>` of type `email` and a label `Email address` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))
- [ ] It has an `<input>` of type `tel` and a label `Phone` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))
- [ ] It has an `<input>` of type `password` and a label `Password` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))
- [ ] It has a `<submit>` button that matches the style of the original page (Blue background, white text) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))